### ISSUE TITLE
berks -h results in null pointer

### ISSUE THREAD

**Issue by 030** [berkshelf/berkshelf#1629]
```
/usr/local/lib/ruby/gems/2.3.0/gems/berkshelf-5.2.0/lib/berkshelf.rb:161:in `const_get': uninitialized constant Berkshelf::Formatter (NameError)
Did you mean?  Berkshelf::NullFormatter
	from /usr/local/lib/ruby/gems/2.3.0/gems/berkshelf-5.2.0/lib/berkshelf.rb:161:in `set_format'
	from /usr/local/lib/ruby/gems/2.3.0/gems/berkshelf-5.2.0/lib/berkshelf/cli.rb:78:in `initialize'
	from /usr/local/lib/ruby/gems/2.3.0/gems/thor-0.19.4/lib/thor.rb:365:in `new'
	from /usr/local/lib/ruby/gems/2.3.0/gems/thor-0.19.4/lib/thor.rb:365:in `dispatch'
	from /usr/local/lib/ruby/gems/2.3.0/gems/berkshelf-5.2.0/lib/berkshelf/cli.rb:52:in `dispatch'
	from /usr/local/lib/ruby/gems/2.3.0/gems/thor-0.19.4/lib/thor/base.rb:444:in `start'
	from /usr/local/lib/ruby/gems/2.3.0/gems/berkshelf-5.2.0/lib/berkshelf/cli.rb:27:in `execute!'
	from /usr/local/lib/ruby/gems/2.3.0/gems/berkshelf-5.2.0/bin/berks:5:in `<top (required)>'
	from /usr/local/bin/berks:22:in `load'
	from /usr/local/bin/berks:22:in `<main>'
```

**Comment by lamont-granquist** [comment]
problem with latest patch release of thor: b0dc41185a2a1420a0491c707c1065884609915b

**Comment by echohack** [comment]
`chef gem install thor` fixes this temporarily while folks are waiting on a new version of ChefDK.

**Comment by echohack** [comment]
@lamont-granquist Is this fix valid? Thor version 0.19.1 isn't available on rubygems, and ChefDK 1.1.16 (latest) ships with thor 0.19.4, which still causes this issue.

**Comment by lamont-granquist** [comment]
0.19.1 exists on rubygems and hasn't been yanked:

https://rubygems.org/gems/thor/versions/0.19.1

IDK why chefdk 1.1.16 is shipping with thor 0.19.4

**Comment by lamont-granquist** [comment]
1.1.16 was released December 15, 2016 and it looks like we must have frozen the berks version at 5.2.0 before shipping so the patch that was only released in berks 5.3.0/5.4.0

new release is coming next week i think (jenkins willing).

**Comment by Zloy** [comment]
+1
